stargz/fs.go

```diff
@@ -312,22 +316,14 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 }
```

Comment:
This comment is meant for line 290 `TODO: bind mount the state directory as a read-only fs on snapshotter's side`

What is `state directory`?

Reply:
State directory is a directory accessible([but the entry is hidden](https://github.com/containerd/stargz-snapshotter/pull/48)) at `/.stargz-snapshotter` in the filesystem, which contains layer metadata(digest, cache status, etc.). See `state` node struct in `stargz/fs.go`.